app/graphql/types/base/field.rb

```diff
@@ -4,10 +4,6 @@ module Types
   module Base
     class Field < GraphQL::Schema::Field
       argument_class Types::Base::Argument
-
-      def resolve_field(obj, args, ctx)
-        resolve(obj, args, ctx)
-      end
```

Comment:
Why you remove this?

Reply:
this was added by the graphql generator, but it causes an error
there is an issue for this in graphql ruby